tests/ui/test_ui_tools.py

```diff
@@ -1270,6 +1272,18 @@ def test_keypress_navigation(self, mocker,
         self.about_view.keypress(size, key)
         super_keypress.assert_called_once_with(size, expected_key)
 
+    def test_feature_level_content(self, mocker, zulip_feature_level):
+        self.controller = mocker.Mock()
+        mocker.patch.object(self.controller, 'maximum_popup_dimensions',
+                            return_value=(64, 64))
+        mocker.patch(VIEWS + '.urwid.SimpleFocusListWalker', return_value=[])
+        about_view = AboutView(self.controller, 'About', zt_version='0.5.1',
+                               server_version='2.2-dev',
+                               server_feature_level=zulip_feature_level)
+        assert len(about_view.feature_level_content) == (
+            1 if zulip_feature_level else 0
+        )
+
```

Comment:
I have added this is an extra test to avoid running all the tests in `TestAboutView` for different feature levels with `zulip_feature_level` fixture (relating to the point about unnecessary extra tests that you mentioned in the review).